.circleci/config.yml

```diff
@@ -40,3 +43,84 @@ jobs:
           command: |
             cd presto-native-execution
             make velox-submodule
+      - run:
+          name: Build
+          command: |
+            source /opt/rh/gcc-toolset-9/enable
+            cd presto-native-execution
+            cmake -B _build/debug -GNinja -DTREAT_WARNINGS_AS_ERRORS=1 -DENABLE_ALL_WARNINGS=1 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=/usr/local -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
+            ninja -C _build/debug -j 8
+      - run:
+          name: 'Run Unit Tests'
+          command: |
+            cd presto-native-execution/_build/debug
+            ctest -j 8 -VV --output-on-failure --exclude-regex velox.*
+      - run:
+          name: 'Maven Install'
+          command: |
+            export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+            ./mvnw clean install ${MAVEN_FAST_INSTALL} -pl 'presto-native-execution' -am
+      - run:
+          name: 'Run e2e tests'
+          command: |
+            export PRESTO_SERVER_PATH="${HOME}/project/presto-native-execution/_build/debug/presto_cpp/main/presto_server"
+            export TEMP_PATH="/tmp"
+            # mvn test ${MAVEN_TEST} -pl 'presto-native-execution' -DPRESTO_SERVER=${PRESTO_SERVER_PATH} -DDATA_DIR=${TEMP_PATH} -Duser.timezone=America/Bahia_Banderas -T1C
+            for test in $(find ${HOME}/project/presto-native-execution/src/test -name 'Test*.java' | grep -o "Test[a-zA-Z]*"); do
+              ./mvnw test ${MAVEN_TEST} -pl 'presto-native-execution' -Dtest=${test} -DPRESTO_SERVER=${PRESTO_SERVER_PATH} -DDATA_DIR=${TEMP_PATH} -Duser.timezone=America/Bahia_Banderas
+            done
+
+  linux-parquet-S3-build:
```

Comment:
We don't need to run linux-parquet-S3-build (and format and header check) every time. We only have to run it if there are changes made to presto-native-execution. See https://github.com/prestodb/presto/blob/master/.github/workflows/test-native-specific.yml.
Also we need to revive MacOS runs on presto-native-execution changes.

Reply:
Is this comment addressed?

Reply:
Not yet, it will be addressed in the next version. Thanks

Reply:
> Is this comment addressed?

Let's keep the jobs here for now before we figure out a way to configure circle to run native-specific jobs.